cloud/services/service/load_balancer.go

```diff
@@ -94,7 +31,8 @@ func (s *Service) GetName(spec *infrastructurev1beta1.OscLoadBalancer) (string,
 		clusterName = infrastructurev1beta1.OscReplaceName(s.scope.GetName())
 		name = clusterName + "-" + "apiserver" + "-" + s.scope.GetUID()
 	}
-	if !ValidateLoadBalancerName(name) {
+	_, err := infrastructurev1beta1.ValidateLoadBalancerName(name)
+	if err != nil {
 		return "", errors.New("Invalid Name")
```

Comment:
return the error maybe (here it is the same)

Reply:
Ok done